bin/schema-registry-run-class

```diff
@@ -26,6 +26,16 @@ for library in "confluent-common" "rest-utils" "schema-registry"; do
   CLASSPATH=$CLASSPATH:$base_dir/share/java/$library/*      
 done
 
+# Log directory to use
+if [ "x$LOG_DIR" = "x" ]; then
+    LOG_DIR="$base_dir/logs"
```

Comment:
I think the danger here is that for the packaged version `$base_dir` will be `/usr`, right? In which case logs would go to `/usr/logs` which is not what most people want.

In other parts of this script we can rely on checking for a file's existence (e.g. https://github.com/coughman/schema-registry/blob/79d3f5fd62865c07e448d1bf1451cb11868e6e7e/bin/schema-registry-run-class#L43) but that doesn't work for this case since it is an output directory.

In Kafka, data directories default to a location in `/tmp` which works across all cases, although isn't ideal for packaged versions. Should we do that and have init scripts override the setting for rpm/deb packages so we can get things in, e.g., `/var/log` instead?